mnist/mlcube.yaml

```diff
@@ -1,11 +1,24 @@
-schema_version: 1.0.0
-schema_type: mlcube_root
-
 name: mnist
-author: MLPerf Best Practices Working Group
-version: 0.1.0
-mlcube_spec_version: 0.1.0
+description: MLCommons MNIST MLCube example
+authors:
+  - {name: "Sergey Serebryakov", email: "sergey.serebryakov@hpe.com", org: "Hewlett-Packard Labs"}
+
+platform:
+  accelerator_count: 0
+
+container:
+  image: mlcommons/mnist:0.0.1        # Image name.
+  build_context: "mnist"              # Docker build context relative to $MLCUBE_ROOT. Default is `build`.
```

Comment:
Let's narrow this down to just "image" for this example. And we solve the build problem in config 2.1. 

Let's make 2.0 focused on the "pull" path. And we look at the build path in 2.1.  This is to unblock Flower and OpenFL.